config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1367,3 +1367,36 @@ periodics:
     testgrid-dashboards: sig-node-kubelet
     testgrid-tab-name: gcp-kubelet-credential-provider
     description: "tests feature gcp kubelet image credential provider"
+- name: ci-cos-inplace-pod-resize-containerd-main-e2e-gce
```

Comment:
maybe note in the name this is a cgroupv2 job.


`ci-cos-cgroupv2inplace-pod-resize-containerd-main-e2e-gce`